### ISSUE TITLE
Broken image in README

### ISSUE THREAD

**Issue by robbyrussell** [orbit-love/orbit-browser-extension#61]
First-time caller and long-time listener. I wanted to mention there is a broken image in your README. 

![2023060548](https://github.com/orbit-love/orbit-browser-extension/assets/257/3800f004-1835-43d0-8dab-a26db0499298)

**Comment by phacks** [comment]
Thanks for flagging @robbyrussell! Long-time user of `ohmyzsh` here, thanks for everything you’re doing there 👋

The image has been fixed, and I’ve learned two unexpected things through this issue:

- Some browsers display the alt text upon encountering a broken image (that’s great!)
- GitHub apparently changed the way it handles paste-in-textarea uploads. If I remember correctly, doing so used to give a URL on the main github assets subdomain (`user-images.githubusercontent.com`)—and that URL was shareable publicly, independently of which repository it was created in. I grabbed the URL from the previous image on a random issue textarea from one of our private repos, and it returned a URL looking like `https://github.com/orbit-love/<a-private-repo>/assets/<id>`. I didn’t give it much thought, but it turns out that this URL _is_ private. So while I (and my colleages) saw the image, the rest of the world didn’t. Looks like I’ll have to pay attention how I generate these URLs from now on!
